P_lin : 1.7
I_lin : 0.04
D_lin : 0.0
P_ang : 3
I_ang : 0.0
D_ang : 0.0
offset_x : 0.4
offset_y : 0
angerr_max : 3
linerr_max : 3
linear_x_max : 0.3
angular_z_max : 0.3
interruption_flag : false
tb_param : "tb3_2"